modules/permutiveRtdProvider.js

```diff
@@ -0,0 +1,171 @@
+/**
+ * This module adds permutive provider to the real time data module
+ * The {@link module:modules/realTimeData} module is required
+ * The module will add custom targeting to ad units of specific partners
+ * @module modules/permutiveRtdProvider
+ * @requires module:modules/realTimeData
+ */
+import { getGlobal } from '../src/prebidGlobal.js'
+import { submodule } from '../src/hook.js'
+import { getStorageManager } from '../src/storageManager.js'
+import { deepSetValue, deepAccess, isFn, mergeDeep } from '../src/utils.js'
+import includes from 'core-js-pure/features/array/includes.js'
+
+export const storage = getStorageManager()
+
+function init (config, userConsent) {
+  return true
+}
+
+/**
+* Set targeting from cache and then try to wait for Permutive
+* to initialise to get realtime targeting
+*/
+export function initTargeting (reqBidsConfigObj, callback, customConfig) {
```

Comment:
Please rename this function to something like "initSegments". Prebid RTD modules uses the word "targeting" to mean ad server targeting -- as a reviewer, I find it confusing to sort out the terminology.